libs/wingsdk/src/testing/assertions.ts

```diff
@@ -0,0 +1,31 @@
+import { InflightClient } from "../core";
+
+/**
+ * The Assert class provides methods for making assertions in tests,
+ * such as comparing two strings for equality.
+ * @inflight
+ */
+export class Assert {
```

Comment:
would you name it `assertions` or `testing`?

Reply:
Is that in lieu of named imports, e.g. `bring { Assert } from testing` or generally a preferences for more granular packages over monolithic packages?

e.g.

```
bring assertions;
bring mocks;
bring testing;
```

vs 

```
bring testing

testing.Assert.equal
...
```

mostly trying to understand the reasoning here and also wondering how this would read best in the code without an alias:

```
bring assertions

assertions.equal
assertions.expectEqual
assertions.isEqual
... ?
```

Reply:
I think `assert` will read best:

```js
bring assert;

assert(x == y);
assert.equals(x, y);
assert.throws(() => {...}, "my error");
```

Can we make this happen?